tests/helpers.py

```diff
@@ -2,22 +2,28 @@
 from pathlib import Path
 from pprint import pprint
 from typing import Optional, List
-from frictionless import validate_package
+from frictionless import validate
+
 
 THIS_SCRIPT_PATH = Path(__file__).parent
 REPOSITORY_ROOT_PATH = THIS_SCRIPT_PATH / ".."
 EXAMPLE_PATH = REPOSITORY_ROOT_PATH / "example" / "datapackage.json"
 PROFILE_PATH = REPOSITORY_ROOT_PATH / "camtrap-dp-profile.json"
-RELAXED_PROFILE_PATH = REPOSITORY_ROOT_PATH / "camtrap-dp-profile-relaxed.json" # File created in later step
+RELAXED_PROFILE_PATH = (
+    REPOSITORY_ROOT_PATH / "camtrap-dp-profile-relaxed.json"
+)  # File created in later step
 TABLE_SCHEMA_PATHS = [
     REPOSITORY_ROOT_PATH / "deployments-table-schema.json",
     REPOSITORY_ROOT_PATH / "media-table-schema.json",
     REPOSITORY_ROOT_PATH / "media-observations-table-schema.json",
-    REPOSITORY_ROOT_PATH / "event-observations-table-schema.json"
+    REPOSITORY_ROOT_PATH / "event-observations-table-schema.json",
 ]
 
-def validate_package_print_and_exit(descriptor_data: dict, paths_to_delete: Optional[List] = None) -> None:
-    report = validate_package(descriptor_data)
+
+def validate_package_print_and_exit(
```

Comment:
Is it a convention to move this to multiple lines? Same for the rest of the changes of this file.